projects/packages/jetpack-mu-wpcom/src/features/starter-page-templates/class-wp-rest-sideload-image-controller.php

```diff
@@ -0,0 +1,295 @@
+<?php
+/**
+ * WP_REST_Sideload_Image_Controller file.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+namespace Automattic\Jetpack\Jetpack_Mu_Wpcom;
+
+/**
+ * Class WP_REST_Sideload_Image_Controller.
+ */
+class WP_REST_Sideload_Image_Controller extends \WP_REST_Attachments_Controller {
+
+	/**
+	 * WP_REST_Sideload_Image_Controller constructor.
+	 */
+	public function __construct() {
+		parent::__construct( 'attachment' );
+
+		$this->namespace = 'fse/v1';
+		$this->rest_base = 'sideload/image';
+	}
+
+	/**
+	 * Register available routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => \WP_REST_Server::CREATABLE,
```

Comment:
Maybe the following part is causing the error?;

```
array(
    // This should have a key
    array(
        'methods' => \WP_REST_Server::CREATABLE,
        'callback' => array($this, 'create_item'),
        'permission_callback' => array($this, 'create_item_permissions_check'),
        'show_in_index' => false,
        'args' => $this->get_collection_params(),
    ),
    'schema' => array($this, 'get_item_schema'),
)
```

the fix should be something like;

```
array(
    'some_key' => array(
        'methods' => \WP_REST_Server::CREATABLE,
        'callback' => array($this, 'create_item'),
        'permission_callback' => array($this, 'create_item_permissions_check'),
        'show_in_index' => false,
        'args' => $this->get_collection_params(),
    ),
    'schema' => array($this, 'get_item_schema'),
)
```

(sorry if I was wrong)